api/build/token.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package build
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-vela/server/internal/token"
+	"github.com/go-vela/server/router/middleware/build"
+	"github.com/go-vela/server/router/middleware/claims"
+	"github.com/go-vela/server/router/middleware/org"
+	"github.com/go-vela/server/router/middleware/repo"
+	"github.com/go-vela/server/util"
+	"github.com/go-vela/types/constants"
+	"github.com/go-vela/types/library"
+	"github.com/sirupsen/logrus"
+)
+
+// swagger:operation GET /api/v1/repos/{org}/{repo}/builds/{build}/token builds GetBuildToken
```

Comment:
I wonder if this endpoint should be in the `api/auth` package instead of `api/build` since it is auth-specific. Then all of the auth stuff is in the same package even if the API spreads it across a variety of endpoints. What do you think?

Reply:
Yeah I was thinking similarly, but landed on sticking with the `build` package since the URL path is very much of the build flavor.